Blockzilla/AppDelegate.swift

```diff
@@ -97,10 +124,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func applicationWillResignActive(_ application: UIApplication) {
         splashView?.animateHidden(false, duration: 0)
+        Telemetry.default.recordSessionEnd()
+        Telemetry.default.recordEvent(TelemetryEvent(category: "action", method: "background", object: "app"))
```

Comment:
Is there a way to change these strings into an `enum`? Specially if there is a limited set of categories and methods, that will prevent typos.